components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -111,6 +112,20 @@ private void insertAccessToken(String accessToken, String consumerKey, AccessTok
                     "Authorized user should be available for further execution.");
         }
 
+        if (OAuthServerConfiguration.getInstance().usePersistedAccessTokenAlias()) {
+            try {
+                accessToken = OAuthServerConfiguration.getInstance().getIdentityOauthTokenIssuer()
```

Comment:
change variable name accessToken to some other name since you change the structure of original value

Reply:
@GayanM would you mind sending a PR for the mentioned improvement?

Reply:
I don't think it is really necessary to change the access token name because if the access token is self-signed only we get the hash as access token, and the hash is stored in the access token column. AFAIK, If we changed the name, we need to call insertAccessToken(accessToken, consumerKey, newAccessTokenDO, connection, userStoreDomain); method also twice.